source/agora/common/Transaction.d

```diff
@@ -90,12 +91,50 @@ public struct Transaction
 
     public void serialize (scope SerializeDg dg) pure const nothrow @safe
     {
+        serializePart(this.inputs.length, dg);
```

Comment:
So this was missing in the serialization part? It should be in its own commit then.

Reply:
Unless @Geod24 approves it.

Reply:
This is added this time.;;